src/main/java/at/hannibal2/skyhanni/config/features/inventory/chocolatefactory/ChocolateFactoryConfig.java

```diff
@@ -171,6 +166,11 @@ public class ChocolateFactoryConfig {
     @FeatureToggle
     public boolean hoppityMenuShortcut = true;
 
+    @Expose
+    @ConfigOption(name = "Rabbit Warning", desc = "")
+    @Accordion
+    public ChocolateFactoryRabbitWarningConfig chocolateFactoryRabbitWarningConfig = new ChocolateFactoryRabbitWarningConfig();
```

Comment:
make class name shorter, rename variable name to "rabbitWarning". "config" should never be part of the variable name, and cf is already implied by the parent class

Reply:
Not sure how to rename it to

Reply:
yeah me neither. lets leave it as is